internal/services/analytics_data_export/datasource_analytics_data_exports.go

```diff
@@ -0,0 +1,281 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package analytics_data_export
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/microsoft/terraform-provider-power-platform/internal/api"
+	"github.com/microsoft/terraform-provider-power-platform/internal/helpers"
+	"github.com/microsoft/terraform-provider-power-platform/internal/services/tenant"
+)
+
+var (
+	_ datasource.DataSource              = &AnalyticsExportDataSource{}
+	_ datasource.DataSourceWithConfigure = &AnalyticsExportDataSource{}
+)
+
+type AnalyticsExportDataSource struct {
+	helpers.TypeInfo
+	analyticsExportClient Client
+}
+
+func NewAnalyticsExportDataSource() datasource.DataSource {
+	return &AnalyticsExportDataSource{
+		TypeInfo: helpers.TypeInfo{
+			TypeName: "analytics_data_exports",
+		},
+	}
+}
+
+func (d *AnalyticsExportDataSource) Metadata(ctx context.Context, req datasource.MetadataRequest, resp *datasource.MetadataResponse) {
+	d.ProviderTypeName = req.ProviderTypeName
+
+	ctx, exitContext := helpers.EnterRequestContext(ctx, d.TypeInfo, req)
+	defer exitContext()
+
+	// Set the proper resource type (common pattern)
+	resp.TypeName = d.FullTypeName()
+	tflog.Debug(ctx, fmt.Sprintf("METADATA: %s", resp.TypeName))
+}
+
+func (d *AnalyticsExportDataSource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+	ctx, exitContext := helpers.EnterRequestContext(ctx, d.TypeInfo, req)
+	defer exitContext()
+	resp.Schema = schema.Schema{
+		MarkdownDescription: "Analytics Data Export configuration",
+		Attributes: map[string]schema.Attribute{
+			"exports": schema.SetNestedAttribute{
+				MarkdownDescription: "The collection of analytics data exports",
+				Computed:            true,
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"id": schema.StringAttribute{
+							MarkdownDescription: "The unique identifier of the analytics data export",
+							Computed:            true,
+						},
+						"source": schema.StringAttribute{
+							MarkdownDescription: "The source of the analytics data",
+							Computed:            true,
+						},
+						"environments": schema.SetNestedAttribute{
+							MarkdownDescription: "The environments configured for analytics data export",
```

Comment:
I don't know if we need orgranisationid's. We work with envid everywhere. I suggest to make environments a set of strings